### DISCUSSION TITLE
Error occurred prerendering page. Invalid revalidate value "[object Object]", must be a non-negative number or "false"

### DISCUSSION THREAD

**Discussion by JonathanBrenner** [vercel/next.js#64999] [Help] [answered]
### Summary

I'm seeing this error since updating next from 13.4.9 to 14.2.x. What's confusing is that I'm clearly assigning it a positive integer value. I'm using the standalone output

### Additional information
Simple reproduction page.tsx

```js
"use client"

export const revalidate = 60 // revalidate this page every 60 seconds

export default function Page() {

    return (
      <div>
        hello world
      </div>
    );
  }

```
```


### Example

_No response_

**Comment by HedTB** [comment]
did u solve ur issue? i just came across the same thing

**Reply by JonathanBrenner** [reply, marked as answer]
Yes I did! Revalidation only works and only makes sense for server components but as you can see in my example I had it in a client component.

**Reply by HedTB** [reply]
Thank you :)

**Reply by xyths** [reply]
> Yes I did! Revalidation only works and only makes sense for server components but as you can see in my example I had it in a client component.

thanks your answer help me
